code/modules/reagents/reagent_containers/glass.dm

```diff
@@ -187,7 +187,7 @@
 	update_icon()
 
 /obj/item/weapon/reagent_containers/glass/bucket
-	desc = "It's a bucket."
+	desc = "This is a bucket. <i>Dear god.</i> There's more. <i>No!</i>"
```

Comment:
What is this? More of what?  This is confusing and makes the forthright description into what I assume is some kind of hackneyed meme insertion.

Reply:
aw man, alright i'll change it